src/status_im/utils/gfycat/core.cljs

```diff
@@ -9,11 +9,19 @@
   [gen vector]
   (str/capitalize (rnd/seeded-rand-nth gen vector)))
 
-(defn generate-gfy
-  ([public-key]
-  (let [gen (rnd/rand-gen public-key)
-        first-adjective (pick-random gen adjectives/data)
+(defn- build-gfy
+  [gen]
+  (let [first-adjective (pick-random gen adjectives/data)
         second-adjective (pick-random gen adjectives/data)
         animal (pick-random gen animals/data)]
     (str first-adjective " " second-adjective " " animal)))
+
+(defn- unknown-gfy "Unknown")
+
+(defn generate-gfy
+  ([public-key]
+   (case public-key
+     nil unknown-gfy
+     "0" unknown-gfy
+     (build-gfy (rnd/rand-gen public-key))))
```

Comment:
Ok, I like this but it would be more REPL friendly if `build-gfy` also accepted public key as the parameter. Basically, if `build-gfy` is the old `generate-gfy`

Reply:
Indeed This is cleaner. Thanks !